yorkie/src/main/kotlin/dev/yorkie/document/time/TimeTicket.kt

```diff
@@ -23,26 +23,6 @@ internal data class TimeTicket(
             ?: delimiter.compareTo(delimiter)
     }
 
-    override fun equals(other: Any?): Boolean {
-        if (this === other) return true
-        if (javaClass != other?.javaClass) return false
-
-        other as TimeTicket
-
-        if (lamport != other.lamport) return false
-        if (delimiter != other.delimiter) return false
-        if (actorID != other.actorID) return false
-
-        return true
-    }
-
-    override fun hashCode(): Int {
-        var result = lamport.hashCode()
-        result = 31 * result + delimiter
-        result = 31 * result + actorID.hashCode()
-        return result
-    }
-
     companion object {
         const val INITIAL_DELIMITER = 0
         const val MAX_DELIMITER = Int.MAX_VALUE
```

Comment:
Not directly related to this PR, but I leave the question below.

The type of delimiter is `uint32`, and the maximum value is also the maximum value of `uint32`. Is there a way to represent a `uint32` in Kotlin?

https://github.com/yorkie-team/yorkie/blob/main/pkg/document/time/ticket.go#L30

Reply:
It's unlikely to happen, but it can cause potential problems, so I recommend using the same type. How about leaving a comment on this?

Reply:
I checked the code created by Java protobuf compiler and it uses `int` for `uint32`.
so... it seems we are better sticking with `Int` for now?

Reply:
Thanks for the check. If so, it would be better to use Int for delimiter in Yorkie.